### PR TITLE
Dev jk

### PR DESCRIPTION
**Pull request by jkamins7**

### CODE REVIEW

File: packages/taxdat/R/file_name_functions.R [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+  all_covariate_choices <- names(covariate_dict)
+  short_covariate_choices <- purrr::map_chr(covariate_dict, "abbr")
+  covariate_choices <- check_covariate_choices(covar_choices = config$covariate_choices,
+                                               available_choices = all_covariate_choices)
+  short_covariates <- short_covariate_choices[covariate_choices]
+  covariate_name_part <- paste(short_covariates, collapse = "-")
+  map_name <- make_map_name(config)
+
+  # Load dictionnary of configuration options
+  config_dict <- yaml::read_yaml(paste0(cholera_directory, "/Analysis/configs/config_dictionnary.yml"))
```
Comment (eclee25) [inline packages/taxdat/R/file_name_functions.R:193]: while we're at it - can we fix the spelling of dictionary here?

File: packages/taxdat/R/mapping_progress_functions.R [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  config <- yaml::read_yaml(config_path)
+  associated_filenames <- get_filenames(config, pipeline_path)
+  dplyr::tibble(
+    stage = names(associated_filenames),
+    filename = associated_filenames
+  ) %>%
+    dplyr::mutate(finished = file.exists(filename)) %>%
+    dplyr::select(-filename) %>%
+    tidyr::pivot_wider(names_from = stage, values_from = finished) %>%
+    mutate(config_path = config_path) %>%
```
Comment (eclee25) [inline packages/taxdat/R/mapping_progress_functions.R:42]: dplyr::mutate

Comment (jkamins7) [reply packages/taxdat/R/mapping_progress_functions.R:42]: should we do a test where we remove the librarying from set_parameters?

Comment (eclee25) [reply packages/taxdat/R/mapping_progress_functions.R:42]: yeah makes sense

**Pull request merged**